cli/schema/cypress.schema.json

```
@@ -124,6 +124,11 @@
       "default": "cypress/plugins/index.js",
       "description": "Path to plugins file. (Pass false to disable)"
     },
+    "screenshot": {
+      "type": "boolean",
+      "default": true,
+      "description": "Whether Cypress will screenshot a failed test run when running headlessly"
```

`headless` and `headed` versus `cypress run` and `cypress open` are commonly mistaken concepts. The screenshots are taken during failed tests during `cypress run` (which could be either in a headed or headless browser). 

```suggestion
      "description": "Whether Cypress will take a screenshot when a test fails during cypress run"
```